packages/keyring-api/src/sol/types.ts

```diff
@@ -0,0 +1,44 @@
+import type { Infer } from '@metamask/superstruct';
+import { array, enums, literal } from '@metamask/superstruct';
+
+import { KeyringAccountStruct, SolAccountType } from '../api';
+import { object, definePattern } from '../superstruct';
+
+/**
+ * Solana addresses are encoded using Base58
+ * Represented as 32 bytes in the format of an Ed25519 PublicKey
+ * And a valid Solana address is 44 characters long
+ */
+export const SolAddressStruct = definePattern(
+  'SolAddress',
+  /^[1-9A-HJ-NP-Za-km-z]{32,44}$/iu,
+);
+
+/**
+ * Supported Solana methods.
+ */
+export enum SolMethod {
+  // General transaction methods
+  SendSolana = 'sendSolana',
```

Comment:
Is this method part of the standard Solana API? The `sendBitcoin` was added because there wasn't really a BTC standard method that we could use, but if there is one for Solana we should use it.